libethash-cuda/CUDAMiner.cpp

```diff
@@ -384,7 +384,7 @@ bool CUDAMiner::cuda_init(
 		if(dagSize128 != m_dag_size || !dag)
 		{
 			// create mining buffers
-			cudalog << "Generating mining buffers"; //TODO whats up with this?
```

Comment:
Could you delete this whole comment (I pushed it years ago on accident)

Reply:
Done